integration-examples/System Center Service Manager/T-SQL Queries/CMDB/Top10UsersWithIssues.sql

```diff
@@ -0,0 +1,32 @@
+-- Displays the top 10 users by incident count raised in the last 30 days
+SELECT TOP 10
+    COUNT(*) as 'Incidents', 
+    usr.DisplayName as Name, 
+    usr.Department_312201FE_C1B3_E95A_01DF_E132E9BD3EC9 as Department,
+    email.Email
+FROM MTV_System$WorkItem$Incident (NOLOCK) incident
+    INNER JOIN RelationshipView affectedUser WITH (NOLOCK) ON incident.BaseManagedEntityId = affectedUser.SourceEntityId 
+        and affectedUser.RelationshipTypeId = 'DFF9BE66-38B0-B6D6-6144-A412A3EBD4CE' -- System.WorkItemAffectedUser
+    INNER JOIN MTV_System$Domain$User usr with (nolock) on affectedUser.TargetEntityId = usr.BaseManagedEntityId
+    LEFT OUTER JOIN(
+        Select 
+            r.SourceEntityId,
+            email.TargetAddress_F96D8DDF_E33A_40B0_4039_F03C3D292F17 as Email
+        from RelationshipView (NOLOCK) r
+            INNER JOIN MTV_System$Notification$Endpoint email WITH (NOLOCK) on r.TargetEntityId = email.BaseManagedEntityId
```

Comment:
That's what you have to do.  I didn't design the SCSM database :)  No not all users will have an endpoint, hence why this is a LEFT OUTER.